src/main/java/pl/javastart/task/DataReader.java

```diff
@@ -0,0 +1,130 @@
+package pl.javastart.task;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.LinkedList;
+import java.util.Queue;
+import java.util.Scanner;
+
+public class DataReader {
+    private static final int EXIT = 0;
+    private static final int ADD_NEW = 1;
+    private static final int  GET_NEXT = 2;
+
+    Scanner scanner = new Scanner(System.in);
+
+    private Queue<Vehicle> vehicles = new LinkedList<>();
+
+    public void chooseOption() {
+        int option;
+        do {
+            printOptions();
+            option = scanner.nextInt();
+            scanner.nextLine();
+            switch (option) {
+                case EXIT:
+                    exit();
+                    break;
+                case ADD_NEW:
+                    addNew();
+                    break;
+                case GET_NEXT:
+                    getNext();
+                    break;
+                default:
+                    System.out.println("Wybrana opcja nie istnieje. Podaj jeszcze raz.");
+            }
+        } while (option != EXIT);
+    }
+
+    public void close() {
+        scanner.close();
+    }
```

Comment:
Nie zamykaj scannera opartego o `System.in`, Scanner może działać również na plikach, wtedy zamykanie ma sens